### PR TITLE
[PAPI-394] Market summary liquidity pool count

### PR DESCRIPTION
**Pull request by drmathias**
Adds the liquidity pool count to market summary, as well as a stored procedure to update it

### CODE REVIEW

File: pipeline-scripts/create-database.sql [hunk 1]
```diff
@@ -952,4 +953,25 @@ DO
...
 
+-- --------
+-- --------
+-- Stored Procedures
+-- --------
+-- --------
+
+CREATE PROCEDURE UpdateMarketSummaryLiquidityPoolCount(IN marketId BIGINT UNSIGNED, IN blockHeight BIGINT UNSIGNED)
+    BEGIN
+        DECLARE _marketId BIGINT UNSIGNED DEFAULT marketId; # clear up ambiguity
```
Comment (drmathias) [inline pipeline-scripts/create-database.sql:964]: DataGrip is showing that `WHERE ms.MarketId = marketId` is always true, as the column name matches the variable name. Redeclared the param with a different name, although thinking we instead use a `_` prefix for all variable names in stored procs.

File: pipeline-scripts/create-database.sql [hunk 2]
```diff
@@ -952,4 +953,25 @@ DO
 DELETE FROM auth_success WHERE Expiry < UTC_TIMESTAMP();
 //
 
+-- --------
+-- --------
+-- Stored Procedures
+-- --------
+-- --------
+
+CREATE PROCEDURE UpdateMarketSummaryLiquidityPoolCount(IN marketId BIGINT UNSIGNED, IN blockHeight BIGINT UNSIGNED)
```
Comment (drmathias) [inline pipeline-scripts/create-database.sql:962]: Need to be able to differentiate between updating all markets (for example during rewinds) and a single market (when a pool is created). There's no concept of optional/default params in MySQL so we have to provide 0 as a market id to the proc if we want to run the update against all pools.

**Review by mrtpain — APPROVED** [review]
lgtm, nice work!

**Comment by drmathias** [comment]
This is now ran against DEV, TEST and MAIN

**Comment by mrtpain** [comment]
This should be merged and not closed right @drmathias?

**Comment by drmathias** [comment]
> This should be merged and not closed right @drmathias? Also ideally going forward, we only run SQL commands one environment at a time as we deploy up.

My bad, I've merged now. That's what I did with this. Merged into dev and checked it was working correctly, then into test and then main

**Pull request merged**